data_collection/web/guides.rst

```diff
@@ -713,3 +713,34 @@ before `trackPageView` was triggered.
 
 We call this procedure a "manual conversion". Manual conversion doesn't cause an additional conversion event to be tracked like the automatic conversion does.
 Automatic conversion tracking requires a "source" event that is analyzed and if it fits some goal definition then it causes an addition conversion event.
+
+Anonymous tracking
+------------------
+
+You can set JavaScript Tracking Client to mark requests to be anonymized. This feature can be useful when you want to use a consent manager on your website and collect full data only from those users who gave consent to be tracked.
```

Comment:
IMO we should not use notion of users and visitors interchangeably. They are not the same:
```suggestion
You can set JavaScript Tracking Client to mark requests to be anonymized. This feature can be useful when you want to use a consent manager on your website and collect full data only from those visitors who gave consent to be tracked.
```